installer.sh

```diff
@@ -303,7 +303,16 @@ else
 		set -e
 		echo "[3/3] No prebuilt PHP found, compiling PHP automatically. This might take a while."
 		echo
-		exec "./compile.sh"
+		logical_cpu_count=$([ $(uname) = 'Darwin' ] && sysctl -n hw.logicalcpu_max ||  lscpu -p | egrep -v '^#' | wc -l) #Get available CPUs to pass to compile.sh and speed up compile
```

Comment:
`egrep` is deprecated, use `grep -e` instead.

```suggestion
		logical_cpu_count=$([ $(uname) = 'Darwin' ] && sysctl -n hw.logicalcpu_max || lscpu -p | grep -e -v '^#' | wc -l) #Get available CPUs to pass to compile.sh and speed up compile
```

Reply:
Also consider using if/else instead of writing support for two different operating systems on one line.